packages/plugins/add/package.json

```diff
@@ -9,11 +9,13 @@
     "build": "codegen-templates-scripts build",
     "pretest": "yarn build"
   },
-  "dependencies": {},
+  "peerDependencies": {
+    "graphql-codegen-core": "^0.13.0"
```

Comment:
@dotansimha peer or direct? When direct dep we might end up like `apollo-link`, instances with the same name but of different versions. We might avoid that because we probably doesn't check anything by `instanceof`.

With peer, we require graphql-codegen-core to be installed, probably with `graphql-code-generator` that has it. I'm not sure it would work though, need to check.

So either direct or peer.

Reply:
Hmm, `peer` ! @kamilkisiela